resources/sass

```diff
@@ -0,0 +1 @@
+../mix/resources/sass
```

Comment:
I guess we could remove this file?

Reply:
Done in a7ce0fa88595b0cbb6d1e67ffcde5587e082efbb.